app/src/main/java/com/beemdevelopment/aegis/ui/PreferencesFragment.java

```diff
@@ -370,6 +374,32 @@ public boolean onPreferenceClick(Preference preference) {
             }
         });
 
+        _pinKeyboardPreference = findPreference("pref_pin_keyboard");
+        _pinKeyboardPreference.setOnPreferenceChangeListener((preference, o) -> {
+            if (!Boolean.parseBoolean(o.toString())) {
+                return true;
+            }
+
+            Dialogs.showPasswordInputDialog(getActivity(), R.string.set_password_confirm, R.string.pin_keyboard_description, password -> {
+                if (android.text.TextUtils.isDigitsOnly(new String(password))) {
```

Comment:
Can you shorten this by adding an import statement for ``android.text.TextUtils``?